src/e2e-app/polyfills.ts

```diff
@@ -9,3 +9,16 @@ if (process.env['ENV'] === 'production') {
   Error['stackTraceLimit'] = Infinity;
   require('zone.js/dist/long-stack-trace-zone');
 }
+
+(function () {
```

Comment:
This code wont be distributed with popover package. @mseemann where should it be put? Maybe in `/src/components/popover/custom-event-polyfil.ts`? There is also npm package with this polyfil... https://www.npmjs.com/package/custom-event-polyfill that could be added as dependency of the popover package.